src/prebid.js

```diff
@@ -519,9 +525,30 @@ function getTargetingfromGPTIdentifier(slot) {
 }
 
 /**
+ * returns targeting keys with key name appended with the bidder code
+ * @param bidArray an array of current bid objects
+ */
+function getTargetingKeysAsBidder(bidArray) {
+  let pairs = {};
+
+  // this assumes no key name collisions, which should not be possible
+  utils._each(bidArray, bid => {
+    const targeting = pb_targetingMap[bid.adUnitCode];
+
+    for (const key in targeting) {
```

Comment:
This also copies the custom keys - not sure we want to do that. An alternative approach would be to iterate over the "standard" keys (`hb*`) stuff instead.